src/screens/Asset/ConfirmRWA.tsx

```diff
@@ -0,0 +1,223 @@
+import SavingIcon from "@assets/icons/saving.svg";
+import { BottomSheetModal } from "@gorhom/bottom-sheet";
+import { Currency } from "@notuslabs/wallet";
+import { Image } from "expo-image";
+import { router } from "expo-router";
+import { useAtom, useAtomValue } from "jotai";
+import { useCallback, useRef, useState } from "react";
+import { View, StyleSheet } from "react-native";
+
+import Container from "@/components/Container";
+import BottomSheet from "@/components/bottomSheet/BottomSheetModal";
+import Button from "@/components/buttons/Button";
+import Divider from "@/components/divider/Divider";
+import InvestmentReceipt from "@/components/receipts/InvestmentReceipt";
+import Heading from "@/components/typography/Heading";
+import Label from "@/components/typography/Label";
+import { borderRadius, dark, base, spacing } from "@/constants/DesignTokens";
+import { SIMPLECURRENCYFORMAT } from "@/constants/configurations";
+import { backedAssets } from "@/constants/mock";
+import useInvestRWA from "@/hooks/useInvestRWA";
+import { amountInputAtom } from "@/lib/atoms";
+import TransactionSuccess from "@/screens/TransactionSuccess";
+
+type ConfirmRWAProps = {
+  asset: string;
+};
+
+type SendTransactionOptions = {
+  currency: Currency;
+  amount: string;
+  asset: string;
+};
+
+export default function ConfirmRWA({ asset }: ConfirmRWAProps) {
+  const [transactionRespose, setTransaciontRespose] = useState<
+    string | undefined
+  >();
```

Comment:
I don't know what happened here, I am sure you've already changed it once, why is it here again?

```suggestion
  const [transactionResponse, setTransactionResponse] = useState<
    string | undefined
  >();
```